src/base58.h

```diff
@@ -146,7 +146,8 @@ template<typename K, int Size, CChainParams::Base58Type Type> class CBitcoinExtK
 
     K GetKey() {
         K ret;
-        ret.Decode(&vchData[0], &vchData[Size]);
+		if (vchData.size() == 74) //if base58 encouded data not holds a ext key, return a !IsValid() key
```

Comment:
I'm not sure that a fixed sized check is a robust solution here (also, hardwiring 74 without any kind of constant) - doesn't the underlying decode function return an error if it couldn't decode?